internal/wait/daemonset.go

```diff
@@ -107,3 +107,12 @@ func (wt Waiter) ForDaemonSetUpdateByKey(ctx context.Context, key ObjectKey) (*a
 	})
 	return updatedDs, err
 }
+
+func (wt Waiter) ForDaemonSetDeleted(ctx context.Context, dskey ObjectKey) error {
+	immediate := true
+	return k8swait.PollUntilContextTimeout(ctx, wt.PollInterval, wt.PollTimeout, immediate, func(aContext context.Context) (bool, error) {
```

Comment:
the preferred generic name for contexts is `ctx`

Reply:
auto typo, thanks for catching.